### DISCUSSION TITLE
[Potential feature] Sorting the topn endpoints

### DISCUSSION THREAD

**Discussion by Superskyyy** [apache/skywalking#11848] [Ideas]
We now have the R3 demo working on the website, one thing to be noticed is that the R3 aggregated endpoints can be (almost always) deep buried in the endpoints until those noises are cleared by the TTL, which could take days to happen. 

One idea is to enable sorting on the endpoints, where the ones with most load always should stay on top. 

Currently, the one with {var} cannot be found unless user explicitly look for it in the searchbox, which is inconvenient. See below:

![image](https://github.com/apache/skywalking/assets/26076517/007c53d6-74da-4c03-90e4-f5c7432a58be)
![image](https://github.com/apache/skywalking/assets/26076517/d0f812bd-aa98-4d6c-a9e5-36ee29c57df6)

 
@kezhenxu94 @wu-sheng What do you think?

**Comment by wu-sheng** [comment]
We were considering about this, the only thing is, elasticsearch performance may be not that good to keep doing TopN for endpoint landing page. 
Reading from endpoint_traffic is just a scanning, but from topN is a kind of complex aggregation query with timerange and entity ID.

**Reply by wu-sheng** [reply]
There is a kind of alternative is to read Endpoint from endpoint_traffic by ording time_bucket desc. If the new grouped endpoint name generated, it should be able to show up, but meanwhile, if you have too many other endpoints not recognized, they are still mixed together.
